### PR TITLE
fix two small bugs - tab error and quotation marks

### PR DESCRIPTION
**Pull request by robochat**
I just found two small errors in the tutorial. One was a tab error!

### CODE REVIEW

File: make.html.markdown [hunk 1]
```diff
@@ -222,7 +222,7 @@ report:
 ifeq ($(sport),tennis)
 	@echo 'game, set, match'
 else
-	@echo 'They think it's all over; it is now'
+	@echo "They think it's all over; it is now"
```
Comment (vendethiel) [inline make.html.markdown]: is that not correct? the echo 2 lines earlier uses single quotes as well

Comment (robochat) [inline make.html.markdown]: The ' in  "it's" closes the quotation and then the ";" finishes the statement and then " it is now" throws an error.

**Comment by robochat** [comment]
These errors in the pattern rules are probably more serious. I thought that this was possible to do but it doesn't work on my current computer and I can no longer find any reference that says that it should work like that. So I've changed to using automatic variables, luckily that section is before the pattern section and so it's a logical progression.

File: make.html.markdown (hunk 1 above)
Comment (vendethiel) [inline make.html.markdown]: oh my, sorry for being blind! good catch

**Comment by vendethiel** [comment]
:+1:

**Pull request merged**